addons/hr/static/src/js/chat.js

```diff
@@ -27,7 +27,9 @@ odoo.define('hr.employee_chat', function (require) {
         },
 
         destroy: function () {
-            this.$el.find('.o_employee_chat_btn').off('click');
+            if (this.el && this.el.querySelector('.o_employee_chat_btn')) {
+                this.el.querySelector('.o_employee_chat_btn').removeEventListener('click', this._onOpenChat);
```

Comment:
I don't think this will actually remove the listener, because `this._onOpenChat` is a different function than `self._onOpenChat.bind(self)` used inside `on` above.

So I would keep the jquery notation here, just ensure `$el` (and `.o_employee_chat_btn` if necessary) is defined before calling it.